traffic_monitor_golang/traffic_monitor/threadsafe/polledcaches.go

```diff
@@ -103,11 +116,22 @@ func copyCaches(a map[enum.CacheName]struct{}) map[enum.CacheName]struct{} {
 	return b
 }
 
+func copyCachesTime(a map[enum.CacheName]time.Time) map[enum.CacheName]time.Time {
+	b := map[enum.CacheName]time.Time{}
+	for k, v := range a {
+		b[k] = v
+	}
+	return b
+}
+
+const PolledBytesPerSecTimeout = time.Second * 60
```

Comment:
It makes me anxious that if we had trouble communicating with a cache which was overloaded with traffic for 10s, we'd tell the Router it's online and has 0 bytes/second and to send it lots of traffic.

It'd take a bit more work to make it a config, that object in the code has no access to the config right now. I'm not a big fan of adding configs for every possible thing, though, every config option is more ops work and training. One doesn't seem like a big deal, but they add up. This should be a rare case; it just happens that we've allowed people to put a dysfunctional 'cache' on our CDN.